services/src/main/java/org/fao/geonet/api/records/MetadataTagApi.java

```diff
@@ -391,7 +391,7 @@ public MetadataProcessingReport tagRecords(
         })
     @ResponseStatus(value = HttpStatus.NO_CONTENT)
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "201", description = "Report about removed records."),
+        @ApiResponse(responseCode = "204", description = "Tag Deleted."),
```

Comment:
Should not be status OK (200)?

Reply:
I matched `@ResponseStatus(value = HttpStatus.NO_CONTENT).`

I did put the note:

> Note: It was noticed that some API's have NO_CONTENT (204) but it is not a void function? But when I tested some they did not seem to return any content.

For now I just wanted to match the @ResponseStatus.

If you want I can create a new issue to be investigated.
 - i.e. Ensure that API's using return code HttpStatus.NO_CONTENT are void functions.

Reply:
Odd - I thought that by changing it to 200 it would require some changes for it to return the report correctly however once I changed it to 200, the report was returned.

So your suggestions have been applied.